pkg/cmd/roachtest/tests/backup_fixtures.go

```diff
@@ -299,15 +310,19 @@ func (bd *backupDriver) monitorBackups(ctx context.Context) error {
 					`PAUSE SCHEDULES WITH x AS (SHOW SCHEDULES) SELECT id FROM x WHERE label = '%s'`, scheduleLabel,
 				)
 				sql.Exec(bd.t, pauseSchedulesQuery)
-				if len(compRunning) > 0 {
-					state = WaitingCompactions
+				if len(compRunning) > 0 || len(backupRunning) > 0 {
+					state = WaitingCompletion
 				} else {
 					state = Done
 				}
 			}
-		case WaitingCompactions:
-			if len(compFailed) > 0 {
-				return errors.Newf("compaction jobs failed: %v", compFailed)
+		case WaitingCompletion:
+			if len(backupFailed) > 0 {
+				return errors.Newf("backup jobs failed in waiting completion: %v", backupFailed)
```

Comment:
"in waiting completion" sounds odd as it is unclear your referring to some phase in the roachtest. To make it clearer, you could say "in WaitingCompletion phase", or you could revise the error to sound like something a human would say :D

Reply:
> sound like something a human would say

I'm just preparing for our AI overlords.